renovate.json

```diff
@@ -1,5 +1,17 @@
 {
   "extends": [
     "github>openfun/renovate-configuration"
+  ],
+  "packageRules": [
+    {
+      "enabled": false,
+      "groupName": "ignored python dependencies",
+      "matchManagers": [
+        "setup-cfg"
+      ],
+      "matchPackageNames": [
+        "django"
+      ]
+    }
```

Comment:
Just FYI, we did it like this in marsha

    {
      "groupName": "allowed django versions",
      "matchManagers": [
        "setup-cfg"
      ],
      "matchPackageNames": [
        "django"
      ],
      "allowedVersions": "<4.1"
    }

I don't know which one you would prefer.

Reply:
I found this syntax really verbose if we have to pin several dependencies nope ?
Furthermore, the `allowedVersions` is currently managed through `setup.cfg`, from my point of view with this syntax we duplicate the logic.

Reply:
Good point, I agree with you
It was only an informative comment :slightly_smiling_face: